### PR TITLE
feat: execute templates on content

### PR DESCRIPTION
**Pull request by bketelsen**
Add the ability to execute a template on the entire dataset, on a single set of models in the dataset, or once per model in a single set of models.

### CODE REVIEW

**Comment by bketelsen** [comment]
imagine adding a -o flag for output filename that had parameterization in it based on the content model:

```blox execute -t article.txt.tmpl -w articles -e -o '[id].go'```

**Comment by bketelsen** [comment]
might consider renaming this to `blox template` because having `blox exec` or `blox execute` with roughly the same format, but instead of running a template, it executes a shell command, would be awesome.

File: cmd/blox_render.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
+/*
```
Comment (rawkode) [inline cmd/blox_render.go:1]: Remove Apache License

File: cmd/blox_render.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+		cobra.CheckErr(err)
+
+		remotes, err := cfg.GetList("remotes")
+		if err == nil {
+			parseRemotes(remotes)
+		}
+
+		pterm.Debug.Printf("\t\tUsing schemata from: %s\n", schemataDir)
+
+		err = filepath.WalkDir(schemataDir, func(path string, d fs.DirEntry, err error) error {
```
Comment (rawkode) [inline cmd/blox_render.go:78]: Is it possible to actually call the build command, instead of duplicating?

**Review by rawkode — CHANGES_REQUESTED** [review]

File: cmd/blox_render.go (hunk 2 above)
Comment (bketelsen) [reply cmd/blox_render.go:63]: I tried to find a way, but couldn't see anything in the cobra docs.

Comment (bketelsen) [reply cmd/blox_render.go:63]: reference #40 - refactor commands to use a central logic structure.

**Pull request merged**